src/services/message.service.ts

```diff
@@ -141,3 +189,8 @@ export class MessageService {
     }
   }
 }
+
+export interface IEmojiTable {
+  emoji: string;
+  emojiValue: any; // This is what you will send to lambda
```

Comment:
does this have to be `any`?

Reply:
Sadly yes, since the idea is you can send it any value with the key.  So I'd you want 1️⃣  to represent 1 you can just put an int, or like here you want it to represent a json object.